.github/workflows/release.yml

```
@@ -3,8 +3,6 @@ name: PyPI Release
 on:
   push:
     branches: [master]
```

```suggestion
    branches: [master]
  pull_request:
    branches: [master]
```
let's keep it and restrict the publish job as it is good to know still the package can be build